test/WebJobs.Script.Tests/DepsFiles/net6.0/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -6,7 +6,7 @@
   "compilationOptions": {},
   "targets": {
     ".NETCoreApp,Version=v6.0": {
-      "Microsoft.Azure.WebJobs.Script.WebHost/4.37.0": {
+      "Microsoft.Azure.WebJobs.Script.WebHost/4.39.100-dev": {
```

Comment:
Only change here (and in the net8.0 deps file was the addition of this new library:

Changed:

Removed:

Added:
- **Microsoft.Azure.Functions.Platform.Metrics.LinuxConsumption.dll: 1.0.0.0/1.0.5.0**